readme.md

```diff
@@ -391,11 +390,68 @@ It's **paramount** that you use one of these two functions so that
 the generated styles can be diffed when the client loads and
 duplicate styles are avoided.
 
+### Asynchronous rendering
+
+`styled-jsx` defaults to using a single, global store for styles that are added
+while rendering. While this works for simple sites (including statically
+rendered pages), it can cause issues if using asynchronous rendering.
+
+To resolve these issues, you can wrap your app in a
+`StyleSheetRegistryContext.Provider` that is unique to the request.
+
+Using NextJS, your `_document.jsx` should look like this:
```

Comment:
Maybe this example can go in a <detail> element so it doesn't take so much space?

Reply:
What do you mean by _in a element_? Like a `<spoiler />` tag?

Reply:
yes I meant <details><summary> ... github cut off `<details>` because I forgot to put it in back ticks :)